hypha/apply/api/v1/screening/tests/test_views.py

```diff
@@ -129,13 +129,15 @@ def test_add_screening_status(self):
             data={'id': self.yes_screening_status.id}
         )
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
-        self.assertEqual(len(response.json()), self.submission.screening_statuses.count())
-        first_submission_screening_status = self.submission.screening_statuses.order_by('-id').last()
+        self.assertEqual(len(response.json()), 2)
+
+        first_submission_screening_status = self.yes_default_screening_status
```

Comment:
I think we need to revert this line. I understand what you are thinking here but we have a unique logic in its [views here](https://github.com/HyphaApp/hypha/blob/main/hypha/apply/api/v1/screening/views.py#L76) that is responsible to change the screening statuses ids of a submission.